TerminalDocs/terminal-chat.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Terminal Chat
+description: Learn how to setup and use Terminal Chat in Windows Terminal Canary.
+author: chrnguyen
+ms.author: chrnguyen
+ms.date: 11/15/2023
+ms.topic: how-to
+---
+
+# Terminal Chat
+Terminal Chat is a feature in [Windows Terminal Canary](https://github.com/microsoft/terminal#installing-windows-terminal-canary) that allows the user to chat with an AI service to get intelligent suggestions (such as looking up a command or explaining an error message) while staying in the context of their terminal.
+
+This feature does not ship with its own large-language model. For now, this feature is only available in Windows Terminal Canary and only supports [Azure OpenAI Service](https://azure.microsoft.com/products/ai-services/openai-service). 
+
+Windows Terminal Canary only communicates with an AI service when the user sends a message in Terminal Chat. The chat history and name of the user’s active shell is also appended to the message that is sent to the AI service. The chat history is not saved by Windows Terminal Canary after their terminal session is over.
+
+> [!NOTE]
+> This feature is only available in [Windows Terminal Canary](https://github.com/microsoft/terminal#installing-windows-terminal-canary).
+
+ ![Terminal Chat UI](./images/terminal-chat.png)
+
+## Setting up Terminal Chat
+ To use Terminal Chat, you will need to add a service endpoint and key to the AI Settings of Windows Terminal Canary. 
+ 
+ For now, Terminal Chat only supports Azure OpenAI Service. To get an Azure OpenAI Service endpoint and key, you will need to create and deploy an Azure OpenAI Service resource.
+
+ ![AI Settings](./images/ai-settings.png)
+
+ ### Creating and Deploying an Azure OpenAI Service resource
+
+ To create and deploy an Azure OpenAI Service resource, please follow the official Azure OpenAI documentation on [creating and deploying an Azure OpenAI Service resource](/azure/ai-services/openai/how-to/create-resource).
+
+In that documentation, you will learn how to:
+
+1. [Create a resource](/azure/ai-services/openai/how-to/create-resource#create-a-resource)
+
+2. [Deploy a model](/azure/ai-services/openai/how-to/create-resource#deploy-a-model)
+
+You will need to use a `gpt-35-turbo` model (or equivalent) with your deployment.
```

Comment:
FYI, I believe the implementation explicitly checks for gpt-35-turbo (so the remark about supporting equivalents wouldn't apply and can be removed).

Reply:
Fixed!